lib/ngx/process.md

```diff
@@ -0,0 +1,191 @@
+Name
+====
+
+`ngx.process` - manage the nginx processes for OpenResty/ngx_lua.
+
+Table of Contents
+=================
+
+* [Name](#name)
+* [Status](#status)
+* [Synopsis](#synopsis)
+    * [Enable privileged agent process and get process type](#enable-privileged-agent-process-and-get-process-type)
+* [Methods](#methods)
+    * [type](#type)
+    * [enable_privileged_agent](#enable_privileged_agent)
+* [Community](#community)
+    * [English Mailing List](#english-mailing-list)
+    * [Chinese Mailing List](#chinese-mailing-list)
+* [Bugs and Patches](#bugs-and-patches)
+* [Author](#author)
+* [Copyright and License](#copyright-and-license)
+* [See Also](#see-also)
+
+Status
+======
+
+This Lua module is currently considered experimental.
+The API is still in flux and may change in the future without notice.
+
+Synopsis
+========
+
+Enable privileged agent process and get process type
+-----------------------------------------
+
+```nginx
+# http config
+init_by_lua_block {
+    local process = require "ngx.process"
+
+    -- enable privileged agent process
+    local ok, err = process.enable_privileged_agent(true)
+    if not ok then
+        ngx.log(ngx.ERR, "enable privileged agent failed error:", err)
+    end
+
+    -- output process type
+    ngx.log(ngx.INFO, "process type: ", process.type(true))
```

Comment:
What's the `true` argument for? I don't think the `process.type` function accepts any arguments.